packages/experiments/src/components/Text/Text.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { IStyle, IPalette, ISemanticColors } from '../../Styling';
+import { IThemedProps } from '../../Foundation';
+import { ITextProps, ITextStyles } from './Text.types';
+
+export const TextStyles = (props: IThemedProps<ITextProps>): ITextStyles => {
```

Comment:
is this the new "standard" for what is called `getStyles` everywhere in OUFR?

Reply:
agree! thanks

Reply:
well.. what about react components?

```tsx
export const TextView = (props: IViewComponentProps<ITextProps, ITextStyles>) => {
```

Reply:
Special case? Keep in mind also, you might decide to change TextStyles from an object to a function. Does that require a rename? Feels bad.